crates/sui-types/src/bridge.rs

```diff
@@ -413,8 +414,12 @@ pub struct BridgeTreasurySummary {
 #[serde(rename_all = "camelCase")]
 pub struct MoveTypeCommitteeMember {
     pub sui_address: SuiAddress,
+    #[schemars(with = "Base64")]
+    #[serde_as(as = "Base64")]
     pub bridge_pubkey_bytes: Vec<u8>,
     pub voting_power: u64,
+    #[schemars(with = "Base64")]
+    #[serde_as(as = "Base64")]
     pub http_rest_url: Vec<u8>,
```

Comment:
I think we want to change this to `String` cuz base64 encoded url will be weird. I took a look at where `MoveTypeCommitteeMember` is referenced in the code now and luckily it doesn't seem very difficult to do. But feel free to leave a TODO here if you don't want to open the can of worm and i will fix it later